src/csvcubed/models/cube/qb/components/concept.py

```diff
@@ -26,6 +26,9 @@ class NewQbConcept(SecondaryQbStructuralDefinition, UriIdentifiable):
 
     def get_identifier(self) -> str:
         return self.code
+    
+    def get_uri_safe_identifier(self) -> str:
```

Comment:
Hang on, this property is already defined on the `UriIdentifiable` type. Let's not override it.